training/amd-bootc/cmd

```diff
@@ -0,0 +1 @@
+make amd FROM=registry.redhat.io/rhel9/rhel-bootc:9.4 DRIVER_TOOLKIT_IMAGE=registry.redhat.io/rhel9/driver-toolkit:5.14.0-427.31.1.el9_4
```

Comment:
I thought we were trying to avoid hard coding a rhel image into this, so that this upstream project can be used by anyone?

Reply:
That's a leftover from tests. I removed it.